flang/lib/Parser/preprocessor.cpp

```diff
@@ -713,6 +820,27 @@ void Preprocessor::Directive(const TokenSequence &dir, Prescanner &prescanner) {
   }
 }
 
+void Preprocessor::PrintMacros(llvm::raw_ostream &out) const {
+  // Sort the entries by macro name.
+  llvm::SmallVector<decltype(definitions_)::const_iterator> entries;
```

Comment:
Use an ordered `std::set<>` and you won't need to sort the result.

Reply:
Done.